scriptmodules/supplementary/skyscraper.sh

```diff
@@ -145,10 +151,10 @@ function _list_systems_skyscraper() {
     find -L "$romdir/" -mindepth 1 -maxdepth 1 -type d -not -empty | sort -u
 }
 
-function remove_skyscraper() {
-    # On removal of the package, purge the cache
-    _purge_skyscraper
-}
+# function remove_skyscraper() {
```

Comment:
please leave the purge on remove operation. If we decide to remove it, then we'll remove the entire function and call, we don't need to comment it.